detectable/src/main/java/com/synopsys/integration/detectable/detectables/sbt/dot/SbtCommandArgumentGenerator.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * detectable
+ *
+ * Copyright (c) 2021 Synopsys, Inc.
+ *
+ * Use subject to the terms and conditions of the Synopsys End User Software License and Maintenance Agreement. All rights reserved worldwide.
+ */
+package com.synopsys.integration.detectable.detectables.sbt.dot;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import com.synopsys.integration.detectable.detectables.sbt.parse.SbtResolutionCacheOptions;
+
+public class SbtCommandArgumentGenerator {
+    @Nullable
+    private final List<String> sbtCommandAdditionalArguments;
+
+    public SbtCommandArgumentGenerator(SbtResolutionCacheOptions options) {
+        if (StringUtils.isNotBlank(options.getSbtCommandAdditionalArguments())) {
```

Comment:
Were there no existing support classes to parse the additional arguments classes? We should look into unifying all those, I know maven has additional args. Unless they need to be handled differently.